bin/compile

```diff
@@ -91,13 +96,18 @@ install_bins() {
     warn_node_engine "$iojs_engine"
     install_iojs "$iojs_engine" "$BUILD_DIR/.heroku/node"
     echo "Using bundled npm version for iojs compatibility: `npm --version`"
+    mcount 'bin.iojs.$(node --version)'
   else
     warn_node_engine "$node_engine"
     install_nodejs "$node_engine" "$BUILD_DIR/.heroku/node"
     install_npm "$npm_engine" "$BUILD_DIR/.heroku/node"
+    mcount "bin.node.$(node --version)"
   fi
   if $YARN; then
     install_yarn "$BUILD_DIR/.heroku/yarn" "$yarn_engine"
+    mcount "bin.yarn.$(yarn --version)"
```

Comment:
fwiw: `node --version` returns a `v` prefixed version, but the other ones don't, which is a little inconsistent. Not sure that matters for you though.